.changeset/nasty-pans-hug.md

```diff
@@ -0,0 +1,11 @@
+---
+'@hashicorp/react-code-block': major
+'@hashicorp/react-docs-page': major
+'@hashicorp/react-hero': major
+'@hashicorp/react-author-byline': minor
+'@hashicorp/react-card': minor
+'@hashicorp/react-marketo-form': minor
+'@hashicorp/react-notification': minor
```

Comment:
The `minor` updates here are pre-1.0 packages, so this should be synonymous with a major per https://semver.org/#spec-item-4

> Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.